.trunk/trunk.yaml

```diff
@@ -0,0 +1,70 @@
+# This file controls the behavior of Trunk: https://docs.trunk.io/cli
+# To learn more about the format of this file, see https://docs.trunk.io/reference/trunk-yaml
+version: 0.1
+cli:
+    version: 1.18.2-beta.14
+# Trunk provides extensibility via plugins. (https://docs.trunk.io/plugins)
+plugins:
+    sources:
+        - id: trunk
+          uri: https://github.com/trunk-io/plugins
+          ref: 3b669ebca634c090a400a989f3a753a33fb09003
+
+# Many linters and tools depend on runtimes - configure them here. (https://docs.trunk.io/runtimes)
+runtimes:
+    enabled:
+        - go@1.21.0
+        - node@18.12.1
+        - python@3.10.8
+tools:
+    runtimes: [node]
+
+# This is the section where you manage your linters. (https://docs.trunk.io/check/configuration)
+lint:
+    definitions:
+        - name: eslint
+          commands:
+              - name: lint
+                run: ${workspace}/bin/eslint.js --output-file ${tmpfile} --format json ${target}
+    enabled:
+        - eslint
+        - actionlint@1.6.26
+        - checkov@3.1.51
+        - git-diff-check
+        - markdownlint@0.38.0
+        - oxipng@9.0.0
+        - prettier@3.1.1
+        - renovate@37.122.0
+        - shellcheck@0.9.0
+        - shfmt@3.6.0
+        - svgo@3.2.0
+        - trivy@0.48.2
+        - trufflehog@3.63.7
+        - yamllint@1.33.0
```

Comment:
Do these versions correspond to the versions in `package.json`? I'm just concerned about needing to update two places when these change.

...or do these have no relation to `package.json` at all?